### PR TITLE
Optimize tidbmonitor readme document.

### PR DESCRIPTION
**Pull request by mikechengwei**
### What problem does this PR solve?
Optimize tidbmonitor readme document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,13 @@ TiDB Operator manages [TiDB](https://github.com/pingcap/tidb) clusters on [Kuber
...
 - __Automatically monitoring TiDB cluster at creating__
 
-    Automatically deploy Prometheus, Grafana for TiDB cluster monitoring, support monitoring multiple clusters, and Thanos framework.
+    Automatically deploy Prometheus, Grafana for TiDB cluster monitoring, support the following features:
+     
+    - Monitoring multiple clusters across multiple namespaces.
+    - Multiple replicas.
+    - Targets sharding.
+    - Configuration and rules dynamic update.
+    - Thanos framework integration.
```
Comment (DanielZhangQD) [inline README.md:51]:
```suggestion
    Automatically deploy Prometheus, Grafana for TiDB cluster monitoring, support the following features:
     
    - Monitoring multiple clusters across multiple namespaces.
    - Multiple replicas.
    - Targets sharding.
    - Updating configurations and rules dynamically.
    - Thanos framework integration.
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by csuzhangxc — APPROVED** [review]

**Comment by mikechengwei** [comment]
@DanielZhangQD The CI can't run.

**Pull request merged**